cmd/cmd.go

```diff
@@ -901,7 +901,7 @@ func NewCLI() *cobra.Command {
 
 	if runtime.GOOS == "windows" {
 		// Enable colorful ANSI escape code in Windows terminal (disabled by default)
```

Comment:
Is this comment about colorful ANSI escape codes still valid? I dont understand how this line was enabling this

Reply:
I can remove it - it was a community contribution.

Reply:
removed